resolwe_bio/tools/tximport_summarize.R

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env Rscript
+library(argparse)
+library(GenomicFeatures)
+library(tximport)
+
+parser = ArgumentParser(description='Summarize TPM values to gene-level expressions.')
```

Comment:
It's usually a good choice to list all output related arguments after all input arguments.